diagnostic.md

```diff
@@ -7,40 +7,48 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone https://github.com/nriser/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git status
```

Comment:
This is a useful command to run in a number of different scenarios, but in this case we want to make sure that we are on a new branch.